braid/braid.c

```diff
@@ -1840,3 +1896,82 @@ braid_SetTimings(braid_Core core,
    return _braid_error_flag;
 }
 
+/*--------------------------------------------------------------------------
+ *--------------------------------------------------------------------------*/
+
+braid_Int
+braid_SetDeltaCorrection(braid_Core           core,      
+                         braid_Int            rank,      
+                         braid_PtFcnInitBasis init_basis,
+                         braid_PtFcnInnerProd inner_prod)
+{
+   /* TODO: compatibility checks */
```

Comment:
I might recommend most other compatibility checks to be done inside braid_Drive (that is, could remove this TODO, and just focus on the one in braid_Drive)